src/components/form/Form.jsx

```diff
@@ -6,9 +6,13 @@ function withWrapper(Component) {
   return class extends React.Component { // eslint-disable-line react/prefer-stateless-function
     render() {
       const { label, ...props } = this.props; // eslint-disable-line react/prop-types
+
+      const required = (props.validations || []).find(e => e == 'required');
```

Comment:
Could you use `===` instead of `==`, please? I'm sure eslint will be happier like this :)